File: Move Zeroes/soln1.java
class Solution {
    public void moveZeroes(int[] nums) {
        int lenNums = nums.length; 
        int j = 0;
        for(int i=0; i<lenNums; i++){
            if(nums[i] != 0 ){
                int temp = nums[j]; 
                nums[j] = nums[i];
                nums[i] = temp;
                j++;
            }
        }
    }
}

File: Running sum of 1d arrays/soln1.py
# Brute force soln

class Solution:
    def runningSum(self, nums: List[int]) -> List[int]:
        newArr = nums
        for i in range(len(nums)):
            temp = nums[i]
            for j in range(0,i):
                temp = nums[i] + nums[j]
            newArr[i] = temp
        return newArr